### DISCUSSION TITLE
discord澳润群组

### DISCUSSION THREAD

**Discussion by snowden-fu** [The-Run-Philosophy-Organization/run#499] [Show and tell]
我创建了一个澳洲润学discord群组，欢迎想润澳的，已经肉身出来但还没拿到身份的，或者打算行动的都可加入
求朋友们三连

https://discord.gg/9qU7bYcM

**Comment by yisabo0312** [comment]
怎么加群啊

**Reply by snowden-fu** [reply]
现在一个人没有，攒多点人再发😭

**Reply by billion-energy** [reply]
> 现在一个人没有，攒多点人再发😭

别啊,发出来人不就多了,俺也目标是润澳,求拉

**Reply by snowden-fu** [reply]
发了，兄弟勿忘三连
> > 现在一个人没有，攒多点人再发😭
> 
> 别啊,发出来人不就多了,俺也目标是润澳,求拉

**Reply by billion-energy** [reply]
> 发了，兄弟勿忘三连
> > > 现在一个人没有，攒多点人再发😭
> > 
> > 别啊,发出来人不就多了,俺也目标是润澳,求拉
> 
> 

Yeah

**Comment by zhukaishi2b** [comment]
过期了 再发下
